src/main/webapp/app/exam/manage/student-exams/student-exam-detail.component.html

```diff
@@ -127,9 +127,13 @@ <h5><span jhiTranslate="artemisApp.studentExams.participationStatus">Participati
                     {{ studentExam.submissionDate | artemisDate: 'long':true }}
                 </li>
                 <li *ngIf="gradingScaleExists">
-                    <strong>{{ isBonus ? ('artemisApp.studentExams.bonus' | artemisTranslate) : ('artemisApp.studentExams.grade' | artemisTranslate) }}:</strong>
+                    <strong>{{ getGradeExplanation() | artemisTranslate }}:</strong>
```

Comment:
Very nice! Using functions is easier to read and gives us type safety in the .ts component file